composer.json

```diff
@@ -44,6 +44,9 @@
         "symfony/assetic-bundle": "~2.3",
         "ezsystems/behatbundle": "^6.1"
     },
+    "conflict": {
+        "doctrine/dbal": "2.7.0"
```

Comment:
Note: This assumes it will be fixed in 2.7.1, is there any WIP on fixing this in 2.7.1?

Reply:
@andrerom: just checked, there's doctrine/dbal#3088, unsure though when they plan 2.7.1 and if it will include it. I'm just hopeful :)

Reply:
Me to, ok lets merge then

Reply:
Thanks @Majkl578 !! ℹ️ 

Especially since our tag with:
```
    "conflict": {
        "symfony/symfony": "3.4.7",
        "doctrine/dbal": "2.7.0"
    },
```


.. did not work... Composer picks the other side of the conflict for some reason 😫 Symfony 3.4.8 is out fixing part of the puzzle now, so 2.7.1 is much appreciated 🏅